exporters/common/src/main/java/io/opentelemetry/exporter/internal/ExporterBuilderUtil.java

```diff
@@ -96,5 +98,95 @@ public static void configureHistogramDefaultAggregation(
     }
   }
 
+  /**
+   * Invoke the {@code aggregationTemporalitySelectorConsumer} with the configured {@link
+   * AggregationTemporality}.
+   */
+  public static void configureOtlpAggregationTemporality(
+      ConfigProperties config,
+      Consumer<AggregationTemporalitySelector> aggregationTemporalitySelectorConsumer) {
+    String temporalityStr = config.getString("otel.exporter.otlp.metrics.temporality.preference");
```

Comment:
These properties are defined [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md) in the context of the OTLP network protocol exporters. Its not clear to me based on today's spec that they would also be applicable to the [proposed `otlp/stdout` exporter](https://github.com/open-telemetry/opentelemetry-specification/pull/4183/files#diff-6e0e0a1cfd634265edad8e3c862f059d3a6f3f61d25fe21df714cfc96764a0b5). For the purposes of the java implementation, let's punt on this until the spec is clear.

If you feel strongly about this, consider extending your spec PR to make it clear that these properties are in fact applicable to `otlp/stdout`. 

This applies to `otel.exporter.otlp.metrics.temporality.preference` and `otel.exporter.otlp.metrics.default.histogram.aggregation`.

Reply:
The spec PR has this to make it clear that these properties are MUST:

https://github.com/open-telemetry/opentelemetry-specification/pull/4183/files#diff-6e0e0a1cfd634265edad8e3c862f059d3a6f3f61d25fe21df714cfc96764a0b5R27-R29